python/tox.ini

```diff
@@ -15,30 +15,35 @@
 #
 
 [tox]
-envlist = py37, py38, py39, py310, flake8, pylint, docs
+envlist = py37, py38, py39, py310, format, lint, docs
 
 [gh-actions]
 python =
     3.7: py37
-    3.8: py38, flake8, pylint
```

Comment:
having `flake8` and `pylint` env split made no sense imo since flake8 also runs safety, mypy etc.
rather have grouping of `lint` and new `format`